metadata.json

```diff
@@ -19,16 +19,324 @@
   ],
   "dependencies": [
     {
-      "name": "simp/compliance_markup",
+      "name": "camptocamp/kmod",
       "version_requirement": ">= 2.1.0 < 3.0.0"
     },
     {
-      "name": "simp/simp_options",
+      "name": "herculesteam/augeasproviders_apache",
```

Comment:
Is there code that you wrote to generate this list? Can you drop it in the Rakefile?

Reply:
@jeefberkey No, there is not